DigitalLearningSolutions.Web/Views/TrackingSystem/Centre/Administrator/_SearchableAdminCard.cshtml

```diff
@@ -2,14 +2,62 @@
 @model SearchableAdminViewModel
 
 <div class="searchable-element nhsuk-panel" id="@Model.Id-card">
-  <details class="nhsuk-details nhsuk-expander nhsuk-u-margin-bottom-0">
-    <summary class="nhsuk-details__summary nhsuk-u-padding-top-0 nhsuk-u-padding-left-0">
-      <span class="nhsuk-details__summary-text searchable-element-title" id="@Model.Id-name" name="name" role="heading" aria-level="2">
+  <details class="nhsuk-details nhsuk-expander">
+    <summary class="nhsuk-details__summary">
+      <span class="nhsuk-details__summary-text searchable-element-title" id="@Model.Id-name" name="name">
         @Model.Name
       </span>
     </summary>
     <div class="nhsuk-details__text">
 
+      <div class="tags">
+        @if (!Model.IsLocked) {
+          <div class="tag nhsuk-u-font-size-14" value="@("IsLocked|false")" hidden>
+            Not locked
+          </div>
+        }
+        @foreach (var (displayText, filter) in Model.Tags) {
+          <div class="tag nhsuk-u-font-size-14" value="@filter">

```

Comment:
Can you use value on divs? I thought it was only a property of HTML inputs..

https://developer.mozilla.org/en-US/docs/Web/API/HTMLInputElement
VS
https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement

(but I might be missing something)

Reply:
Technically no but also kind of yes. 
`document.getElementById("id").value;` won't work for this since it isn't a property of a div
But there isn't really anything to stop you putting random attributes on a div and using `document.getElementById("id").getAttribute('value');`

I guess I should probably just name it something else so it isn't confusing. e.g. filter-value

Reply:
Changed to data-filter-value